apps/condo/domains/ticket/components/BaseTicketForm/index.tsx

```diff
@@ -142,7 +142,7 @@ export const TicketInfo = ({ form, validations, UploadComponent, initialValues,
                                             currentLength={currentDetailsLength}
                                             maxLength={500}
                                             onChange={e => setCurrentDetailsLength(e.target.value.length)}
-                                            placeholder={DescriptionPlaceholder}
+                                            placeholder={DescriptionPlaceholder + ' *'}
```

Comment:
Looks like 

" Description * "

And not like 

" * Description "

![image](https://user-images.githubusercontent.com/1640424/159419151-6919fc11-56f8-4983-b556-1041abd4eaa6.png)